template/js/index.js

```diff
@@ -68,11 +68,13 @@ require('nighthawk')({
                   <h3><a href="${config.baseUrl}/issues/${tag.name}">${tag.name}</a></h3>
                   <ul>
                     ${issues.map((issue) => {
+                      const repoOwnerUrl = `https://www.github.com/${issue.project.repoOwner}`;
+                      const repoUrl = `${repoOwnerUrl}/${issue.project.repo}`;
                       return html`
                           <li>
                             <span class="project-link">
-                              <a href="https://www.github.com/${issue.project.repoOwner}" target="_blank">${issue.project.repoOwner}</a>
```

Comment:
Would you mind keeping it inline?  In the future I think we could componentize some of this better (this was just a hack job to get started lol), but until then I would like to keep consistent.  Since all the other parts are not pulled out as variables, I would rather keep consistent even if it means repeating the `https://www.github.com/` part.